app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -138,7 +138,7 @@ class AdministratorControlsActivityTest {
   }
 
   @Test
-  fun testAdministratorControlsFragment_loadFragment_displayGeneralAndProfileManagement() {
+  fun testAdministratorControlsFragment_displayGeneralAndProfileManagement() {
```

Comment:
`testAdministratorControlsFragment_generalAndProfileManagementIsDisplayed()`

Reply:
Thanks for review sir, I will do so.

Reply:
Done, PTAL.